bip-taproot.mediawiki

```diff
@@ -253,6 +253,7 @@ Using fresh keys implies that taproot output construction does not need to take
 This does not avoid leaking information through the leaf depth and therefore only applies to balanced (sub-) trees.
 In addition, every leaf should have a set of keys distinct from every other leaf.
 The reason for this is to increase leaf entropy and prevent an observer from learning an undisclosed script using brute-force search.
+In the unlikely case that a leaf does not contain keys or the keys have to be reused, the tapscript can be salted by adding a push of 16 bytes of random data to the stack, followed by ''OP_DROP''.
```

Comment:
I'm not sure about advising this. If someone is willing to go this length in order to get distinct addresses, they could just as well use different keys.

Reply:
Okay, removed that part.